qa/suites/upgrade/jewel-x/ceph-deploy/jewel-luminous.yaml

```diff
@@ -0,0 +1,61 @@
+meta:
+- desc: |
+    Setup 4 node ceph cluster using ceph-deploy, use latest
+    stable jewel as initial release, upgrade to luminous and
+    also setup mgr nodes along after upgrade, check for
+    cluster to reach healthy state, After upgrade run kernel tar/untar
```

Comment:
may be to use full stop after `healthy state`

Reply:
I feel comma is still appropriate there since its explaining the full test.